snuba/cli/__init__.py

```diff
@@ -61,7 +61,7 @@ def get_command(self, ctx: Any, name: str) -> click.Command:
                 code = compile(f.read(), fn, "exec")
                 eval(code, ns, ns)
             init_time = time.perf_counter() - start
-            metrics.gauge("snuba_init", init_time)
+            metrics.timing("snuba_init", init_time)
```

Comment:
You might want to have a different name. I'm not sure how DD handles a metric with the same name but two different types.